examples/helloworld/WORKSPACE

```diff
@@ -0,0 +1,27 @@
+workspace(name = "examples_helloworld")
+
+# If copying this WORKSPACE file to start a new Bazel workspace, replace this with a
+# `http_archive` repository rule as specified here: https://github.com/bazelbuild/rules_python/releases
+local_repository(
+    name = "rules_python",
+    path = "../../",
+)
+
+load("@rules_python//python:repositories.bzl", "py_repositories")
+
+py_repositories()
```

Comment:
this doesn't do anything, and I think if rules_python were to grow dependencies it would be horribly breaking because WORKSPACE deps semantics are so bad, users will have to re-order their WORKSPACE if they can get it to work at all. WDYT about removing this notion that rules_python always makes you install additional runtime bazel deps? we should get what we need from python packages on pypi.

Reply:
Yep, I've seen your bazelcon talk and I'm onboard 😅